consensus/fsm_handler.go

```diff
@@ -0,0 +1,111 @@
+package consensus
+
+import (
+	"fmt"
+
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	"github.com/pokt-network/pocket/shared/codec"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+func (m *consensusModule) HandleStateTransitionEvent(transitionMessageAny *anypb.Any) error {
+	m.m.Lock()
+	defer m.m.Unlock()
+	m.logger.Info().Msgf("Received a state transition message: %s", transitionMessageAny)
```

Comment:
Given your other in flight PR, we can probably improve this, right?

Reply:
Correct, added the comment.